app/src/main/assets/extensions/fxr_youtube/background.js

```diff
@@ -33,6 +33,12 @@ function redirect(details) {
     })
 
     if (found) {
+        // We should not add the app param to the timedtext requests (captions),
+        // otherwise they will fail.
+        if (uri.href.includes("timedtext")) {
```

Comment:
Nice finding!

I'd check the uri against "http://youtube.com/api/timedtext" tough, seems safer than just checking the existence of timedtext (although I agree should work in 99% of the cases"

Reply:
Let's use '.com/api/timedtext?' to match then. The host name can be "https://*.youtube.com/*" and "https://*.youtube-nocookie.com/*", so this should be general enough.